bundles/org.eclipse.cdt.lsp/src/org/eclipse/cdt/lsp/ui/navigator/LspCEditorOpenActionProvider.java

```diff
@@ -70,6 +87,25 @@ public IWorkbenchPage getPage() {
 
 	private OpenCFileAction openCFileAction;
 	private IViewPart viewPart = null;
+	
+	public static void revealInEditor(IEditorPart part, DocumentSymbolWithFile element) {
```

Comment:
Not clear why this method is `public static` and placed in this type, that creates circular referencing between `OpenCFileAction` and `LspCEditorOpenActionProvider`

Reply:
Thanks, your're right I'll check that!